src/pages/council.js

```diff
@@ -41,30 +41,24 @@ const CouncilPage = () => {
             <MuiLink href="mailto:hans-johnson@uiowa.edu">Hans Johnson</MuiLink>{' '}
             - The University of Iowa (Treasurer)
           </li>
-          <li>
-            <MuiLink href="mailto:pauly2@mail.med.upenn.edu">
-              Paul Yushkevich
-            </MuiLink>{' '}
-            - University of Pennsylvania (Chief Technology Officer)
-          </li>
         </ul>
 
-        <p>Addition Council members are:</p>
+        <p>Addition Council Members are:</p>
         <ul>
           <li>Andras Lasso - Queen's University</li>
-          <li>
-            Jon Haitz Legarreta Gorroño - PhD Student, Université de Sherbrooke
-          </li>
-          <li>
-            Lauren O'Donnell - Brigham and Women's Hospital - Harvard Medical
-            School
-          </li>
+          <li>Jon Haitz Legarreta Gorroño - Postdoctoral Research Fellow, Brigham and Womens Hospital, Harvard Medical School</li>
```

Comment:
Ah, yes, thanks Matt :-). A typo in the name of the hospital: Brigham and Women's Hospital

Reply:
Thanks for the review @jhlegarreta, addressed